### ISSUE TITLE
Unit that parachutes unconscious is in awake animation after landing

### ISSUE THREAD

**Issue by commy2** [acemod/ACE3#691]
commit 54daa02aa4dbd216d497eeba5d8955024eeeaa36

CBA, ACE

```
player addBackpack "ACE_NonSteerableParachute";
player setPos (getPos player vectorAdd [0,0,500]);

[player, true] call ace_medical_fnc_setUnconscious;
```

Works fine while falling, but when landing if unconscious the unit is in the awake animation.

**Comment by commy2** [comment]
This is the work around we used in AGM:
https://github.com/KoffeinFlummi/AGM/blob/master/AGM_Medical/functions/fn_knockOut.sqf#L83-L98

It used spawn, so... Player is treated like being on foot , when being inside parachute. The issue is, I belive thaat the player never actually get's out of the vehicle (getOut event), but the parachute get's deleted. 

(The issue might therefore be reproduceable by deleting any vehicle an unconscious AI is in, IDK)

**Comment by commy2** [comment]
Also note, that this has to be debugged with and without ACE_Parachuting, because there are some aniation tweaks in there which might change things.

**Comment by Glowbal** [comment]
Fixed in #717 
Thanks for the repo code @commy2 Made it really easy to test the fix. :)
